### ISSUE TITLE
equivalent of PyTorch `data`

### ISSUE THREAD

**Issue by yueyinqiu** [dotnet/TorchSharp#1305]
Is there an equivalent of PyTorch `data`?

```python
import torch

t = torch.zeros([], requires_grad=True)
# maybe some parameter in a module
# which makes it unable to use t.fill_ because it requires grad

t.data.fill_(1)
print(t)
```

**Comment by yueyinqiu** [comment]
We can and should always use `detach` instead.

https://discuss.pytorch.org/t/the-difference-between-data-and-detach/30926/2
